### DISCUSSION TITLE
FR: Utilities → Project Config user permission

### DISCUSSION THREAD

**Discussion by mattstein** [craftcms/cms#9692] [Ideas] [answered]
The current _Utilities_ lineup sports the following:

<img width="777" alt="Screen Shot 2021-08-09 at 12 17 24 PM@2x" src="https://user-images.githubusercontent.com/2488775/128746945-cf2cfaa5-fa3b-48d4-b662-006e54623d20.png">

I noticed recently that there isn’t the option to grant someone access to the Project Config utility, and my first thought was that it was possibly omitted since _maybe_ only an admin should be able to apply (or discard) project config updates that may have an important impact on the site.

But since _Migrations_ are on that list and could have just about the same impact, it makes sense that _Project Config_ should be an available permission.

**Comment by brandonkelly** [comment, marked as answer]
I don’t remember why exactly, but the utility is very intentionally exclusive to admins, and has been since it was added:

https://github.com/craftcms/cms/blob/754df2b84742a4fdf2faa6540463af28d02c1151/src/services/UserPermissions.php#L624-L627

Guessing it’s because it’s potentially pretty dangerous.

**Reply by mattstein** [reply]
More dangerous than migrations, though?
